README.md

```diff
@@ -1 +1,45 @@
-# design-teardown
\ No newline at end of file
+# design-teardown
+This project consists of building a heatmap of the Smashing magazine website, for to understand the UX design
+
+# Original Page
+<p align="center">
+  <img src="img/screenshop.png" width="850" title="screenshot">
+</p>
+
+
+Built With : Html,CSS and github technologies used:flex-box and grid of CSS, Using StyleLinters.
+
+Live Demo Link:https://rawcdn.githack.com/Fanger53/design-teardown/25fdf1bccda29d75853e9fd849c8cfa9f81042ad/index.html
```

Comment:
- [ ] To have a professional readme, please update your link, it is displaying your page without the changes. As a recommendation, so you don't have to be manually updating this link during your code reviews, use the second link (development link) in Githack, that one will automatically load your last version in the same branch :)